Cant build it

### Describe the bug

org.apache.maven.project.ProjectBuildingException: Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for org.keycloak.quickstarts:event-listener-sysout:22.0.1: Failure to find org.keycloak.quickstarts:keycloak-quickstart-parent:pom:22.0.1 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 20, column 13


### Version

22.0.1

### Expected behavior

I am using ubuntu for building

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Anything else?

_No response_

@pedroigor After recent changes I have also problems to build QS.
```
mvn -f extension/user-storage-simple/ clean install
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for org.keycloak:keycloak-server-spi:jar is missing. @ line 50, column 21
[ERROR] 'dependencies.dependency.version' for org.keycloak:keycloak-model-legacy:jar is missing. @ line 55, column 21
[ERROR] 'dependencies.dependency.version' for org.jboss.logging:jboss-logging:jar is missing. @ line 60, column 21
[ERROR] 'dependencies.dependency.version' for org.jboss.arquillian.graphene:graphene-webdriver:pom must be a valid version but is '${arquillian-graphene.version}'. @ line 68, column 22
[ERROR] 'dependencies.dependency.version' for org.jboss.arquillian.extension:arquillian-phantom-driver:jar is missing. @ line 72, column 21
[ERROR] 'dependencies.dependency.version' for org.jboss.arquillian.graphene:arquillian-browser-screenshooter:jar is missing. @ line 77, column 21
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project org.keycloak.quickstarts:keycloak-user-storage-properties:22.0.1 (/home/mkanis/workspace/keycloak-quickstarts/extension/user-storage-simple/pom.xml) has 6 errors
[ERROR]     'dependencies.dependency.version' for org.keycloak:keycloak-server-spi:jar is missing. @ line 50, column 21
[ERROR]     'dependencies.dependency.version' for org.keycloak:keycloak-model-legacy:jar is missing. @ line 55, column 21
[ERROR]     'dependencies.dependency.version' for org.jboss.logging:jboss-logging:jar is missing. @ line 60, column 21
[ERROR]     'dependencies.dependency.version' for org.jboss.arquillian.graphene:graphene-webdriver:pom must be a valid version but is '${arquillian-graphene.version}'. @ line 68, column 22
[ERROR]     'dependencies.dependency.version' for org.jboss.arquillian.extension:arquillian-phantom-driver:jar is missing. @ line 72, column 21
[ERROR]     'dependencies.dependency.version' for org.jboss.arquillian.graphene:arquillian-browser-screenshooter:jar is missing. @ line 77, column 21
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
```

--- a/extension/action-token-authenticator/README.md
+++ b/extension/action-token-authenticator/README.md
@@ -46,7 +46,7 @@ Build and Deploy the Quickstart
 To build the provider, run the following maven command:
 
    ````
-   mvn clean install
+   mvn -Pextension -DskipTests clean install
    ````
 
 To install the provider, copy the `target/action-token-example.jar` JAR file to the `providers` directory of the server distribution.
@@ -85,9 +85,9 @@ Integration test of the Quickstart
 1. Make sure you have an Keycloak server running with an admin user in the `master` realm or use the provided docker image. Also make sure that server
 was started with the parameters as described above 
 2. You need to have Chrome browser installed and updated to the latest version
-3. Run `mvn clean install -Djakarta`
+3. Run `mvn -Pextension clean install`
 
-After running the above command, the WAR file will be located in
+After running the above command, the WAR file to deploy the external service will be located in
 `target/deployments/wildfly_action-token-responder-example_action-token-responder-example.war`.
 
 Among other things, the test did 2 things:

--- a/extension/action-token-required-action/README.md
+++ b/extension/action-token-required-action/README.md
@@ -45,7 +45,7 @@ Build and Deploy the Quickstart
 To build the provider, run the following maven command:
 
    ````
-   mvn clean install
+   mvn -Pextension clean install
    ````
 
 To install the provider, copy the `target/action-token-req-action-example.jar` JAR file to the `providers` directory of the server distribution.
@@ -84,7 +84,7 @@ Integration test of the Quickstart
 1. Make sure you have an Keycloak server running with an admin user in the `master` realm or use the provided docker image. Also make sure that server
    was started with the parameters as described above
 2. You need to have Chrome browser installed and updated to the latest version
-3. Run `mvn clean install -Djakarta`
+3. Run `mvn -Pextension clean install`
 
 After running the above command, the WAR file will be located in
 `target/deployments/wildfly_action-token-responder-example_action-token-responder-example.war`.

--- a/extension/event-listener-sysout/README.md
+++ b/extension/event-listener-sysout/README.md
@@ -30,7 +30,7 @@ Build and Deploy the Quickstart
 To build the provider, run the following maven command:
 
    ````
-   mvn clean install
+   mvn -Pextension clean install
    ````
 
 To install the provider, copy the target/event-listener-sysout.jar JAR file to the `providers` directory of the server distribution.

--- a/extension/event-store-mem/README.md
+++ b/extension/event-store-mem/README.md
@@ -43,7 +43,7 @@ Build and Deploy the Quickstart
 To build the provider, run the following maven command:
 
    ````
-   mvn clean install
+   mvn -Pextension clean install
    ````
 
 To install the provider, copy the target/event-store-mem.jar JAR file to the `providers` directory of the server distribution.

--- a/extension/extend-account-console/README.md
+++ b/extension/extend-account-console/README.md
@@ -38,7 +38,7 @@ Configuration in <span>Keycloak</span>
 To build the provider, run the following maven command:
 
    ````
-   mvn clean install
+   mvn -Pextension clean install
    ````
 
 To install the provider, copy the `target/keycloak-man-theme.jar` JAR file to the `providers` directory of the server distribution.

--- a/extension/user-storage-jpa/README.md
+++ b/extension/user-storage-jpa/README.md
@@ -36,7 +36,7 @@ you make sure the persistence unit will be using the correct datasource.
 To build the provider, run the following maven command:
 
    ````
-   mvn clean install
+   mvn -Pextension clean install
    ````
 
 To install the provider, copy the target/user-storage-jpa-example.jar JAR file to the `providers` directory of the server distribution.

--- a/extension/user-storage-simple/README.md
+++ b/extension/user-storage-simple/README.md
@@ -42,7 +42,7 @@ Build and Deploy the Quickstart
 To build the provider, run the following maven command:
 
    ````
-   mvn clean install
+   mvn -Pextension clean install
    ````
 
 To install the provider, copy the target/user-storage-properties-example.jar JAR file to the `providers` directory of the server distribution.

